test/__snapshots__/mathml-spec.js.snap

```diff
@@ -347,7 +347,7 @@ exports[`A MathML builder should output \\limsup_{x \\rightarrow \\infty} correc
             ∞
           </mi>
         </mrow>
-      </munder>
+      </msub>
```

Comment:
When should we be using `munder` vs. `msub`?

Reply:
The `<munder>` was a hack because we did not have a way to apply `\limits` to `\operatorname`. Now that we can apply `\limits`, we should expect to see `<msub>` in the snapshot of a text style example.